packages/demo-protobuf/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "baseUrl": ".",
+    "outDir": "build",
+    "declarationDir": "build/types",
+    "experimentalDecorators": true,
```

Comment:
Looking at `packages/demo-protobuf/build/decorator.js`, TypeScript seems to have its own compatibility implementation, such that core-js is not required.